docs/source/contributing.rst

```diff
@@ -90,12 +92,17 @@ Submit a request::
                                      --amount 1 \
                                      --target-address 0x1CEE82EEd89Bd5Be5bf2507a92a755dcF1D8e8dc \
                                      --target-chain-id 1337 \
-                                     --target-token-address $TOKEN_ADDR
+                                     --target-token-address $TOKEN_ADDR \
+                                     --validity-period 2700
 
 
 Working with a local Optimism instance
 --------------------------------------
 
+For MacOS add an alias to your bash for mktemp as follows::
+
+    alias mktemp=gmktemp
```

Comment:
Eh, it would be nice if our scripts did not require that. And I think @GabrielBuragev already had a fix that avoid the GNU-specific, option, is that right, Gabriel?

Reply:
Yes, exactly. That fix can be applied on the code level. 
Should i submit it as a separate PR or do you want to include it in this PR?

The change is the following:

change [this](https://github.com/beamer-bridge/beamer/blob/main/docker/optimism/optimism.sh#L33) line to: 
`WAIT_FOR_SEQUENCER_SCRIPT=$(mktemp)-wait-for-sequencer.sh`

Reply:
Yes, @GabrielBuragev, I prefer your fix here, please submit a PR and then we can drop this change.